docs/application/dotnet/guides/app-management/component-port.md

```diff
@@ -0,0 +1,466 @@
+
+            public int Sequence
+            {
+                get; set;
+            }
+        }
+
+        [Serializable]
+        public class Response
+        {
+            public Response(string command, int sequence, int result)
+            {
+                Command = command;
+                Sequence = sequence;
+                Result = result;
+            }
+
+            public string Command
+            {
+                get; set;
+            }
+
+            public int Sequence
+            {
+                get; set;
+            }
+
+            public int Result
+            {
+                get; set;
+            }
+        }
+
+        public class ServerService : ServiceComponent
+        {
+            private static string LogTag = "ServerService";
+            private ComponentPort _port;
+            private Thread _thread;
+
+            private void OnRequestedReceived(object sender, RequestEventArgs args)
+            {
+                Log.Info("Sender: " + args.Sender);
+                var request = args.Request;
+                if (request.GetType() == typeof(Request))
+                {
+                    var req = (Request)request;
+                    Log.Info("Command: " + req.Command);
+                    Log.Info("Sequence: " + req.Sequence);
+                    Log.Info("Message: " + req.Message);
+
+                    if (args.IsReplyRequested)
+                    {
+                        args.Reply = new Response(req.Command, req.Sequence, 0);
+                    }
+                }
+            }
+
+            private void OnThread()
+            {
+                Log.Info("OnTrhead()");
+                _port.WaitForEvent();
+            }
+
+            public override override OnCreate()
+            {
+                Log.Info("OnCreate()");
+                base.OnCreate();
+                _port = new ComponentPort("ServerService");
+                _port.RequestReceived += OnRequestReceived;
+                _thread = new Thread(new ThreadStart(OnThread));
+                _thread.Start();
+                return true;
+            }
+
+            public override void OnStartCommand(AppControl appControl, bool restarted)
+            {
+                Log.Info("OnStartCommand()");
+                base.OnStartCommand(appControl, restarted);
+            }
+
+            public override void OnDestroy()
+            {
+                Log.Info("OnDestroy()");
+                base.OnDestroy();
+
+                if (_port != null)
+                {
+                    _port.RequestReceived -= OnRequestReceived;
+                    _port.Dispose();
+                    _port = null;
+                }
+            }
+        }
+    }
+    ```
+
+3.  Set up the receiving thread of the component.
+
+    1. To have the receiving thread of the component wait for incoming requests, call `WaitForEvent()` of the `Tizen.Applications.ComponentBased.ComponentPort` class.
```

Comment:
@nawab-samsung, @hjhun, is this sequential? An ordered list is needed here?

Reply:
It's not sequential. The purpose of numbering is for readability.
If you think it's not needed, please tell me.

Reply:
@hjhun, if these steps are not to be performed in an order (1.2.3) as listed, kindly change to unordered list